Composer/packages/adaptive-form/src/components/fields/IntellisenseFields.tsx

```diff
@@ -161,9 +165,10 @@ export const IntellisenseNumberField: React.FC<FieldProps<string>> = (props) =>
       onBlur={props.onBlur}
       onChange={onChange}
     >
-      {({ textFieldValue, focused, onValueChanged, onKeyDownTextField, onKeyUpTextField, onClickTextField }) => (
+      {({ textFieldValue, focused, onValueChanged, onKeyDownTextField, onKeyUpTextField, onClickTextField, aria }) => (
         <NumberField
           {...props}
+          aria={aria}
           focused={focused}
           id={id}
           value={textFieldValue}
```

Comment:
There is also a `JsonField` wrapped into `Intellisense` down the lines. Verified that it doesn't make sense to add `aria` there:
![image](https://user-images.githubusercontent.com/2841858/161605395-adb07e06-994f-4d9d-a20c-c859a285596b.png)

- It renders Monaco editor which has it's own more complicated accessibility handling
- We don't provide any suggestions for the JSON field (AFIK) - only the placeholder message
- If we really want to provide suggestions for the JSON field, we should use Monaco built-in autocomplete feature as it is more accessible and user-friendly